engine/src/main/resources/org/hibernate/validator/ValidationMessages_pt_BR.properties

```diff
@@ -15,13 +15,15 @@ javax.validation.constraints.Size.message        = tamanho deve estar entre {min
 
 org.hibernate.validator.constraints.CreditCardNumber.message    = N\u00FAmero de Cart\u00E3o de Cr\u00E9dito inv\u00E1lido
 org.hibernate.validator.constraints.Email.message               = N\u00E3o \u00E9 um endere\u00E7o de e-mail
+org.hibernate.validator.constraints.EAN.message                 = C�digo de barras {type} inv\u00E1lido
```

Comment:
Perhaps it's a good idea to replace the accented char with its unicode representation? Just to avoid file encoding conflicts